lib/utils/shorten-url.ts

```diff
@@ -0,0 +1,12 @@
+export async function shortenUrl(url: string) {
+  try {
+    const response = await fetch(`${process.env.NEXT_PUBLIC_API_URL}/url/shorten?url=${encodeURIComponent(url)}`);
+
+    if (response.ok) {
+      const data = (await response.json()) as { shortUrl: string };
+      return data.shortUrl;
+    }
+  } catch (e) {}
```

Comment:
consider logging an explicit error to Sentry if URL shortening doesn't work. Also, if URL shortening fails, should we fallback to the full URL?

Reply:
Good idea about Sentry.

And yes, if the request fails or the API is down, it falls back to the original URL